code/modules/mob/living/carbon/life.dm

```diff
@@ -266,7 +266,7 @@
 	if(breath_gases[/datum/gas/miasma])
 		var/miasma_partialpressure = (breath_gases[/datum/gas/miasma][MOLES]/breath.total_moles())*breath_pressure
 
-		if(prob(1 * miasma_partialpressure))
+		if(prob(0.2 * miasma_partialpressure))
```

Comment:
why are you touching miasma?

Reply:
From what I know, miasma is causing most of the issues.

Reply:
you should read the pr that made these changes closer, because it's not